etc/zoom.profile

```diff
@@ -27,7 +27,7 @@ nodvd
 nonewprivs
 noroot
 notv
-protocol unix,inet,inet6
-seccomp
+protocol unix,inet,inet6,netlink
+seccomp !chroot
```

Comment:
can someone please explain what the change to line 31 means? is it disabling some aspects of seccomp?

Reply:
It allows the chroot syscall.